oqs/wrapper.py

```diff
@@ -193,12 +173,49 @@ def free(self):
     def __repr__(self):
         return "Key encapsulation mechanism: " + self._kem.contents.method_name.decode()
 
+
+liboqs.OQS_KEM_new.restype = ct.POINTER(OQS_KEM)
+liboqs.OQS_KEM_alg_identifier.restype = ct.c_char_p
+
+
+def is_KEM_enabled(alg_name):
+    """Returns True if the KEM algorithm is enabled, False otherwise.
+
+    Attribute:
+        alg_name -- a KEM mechanism algorithm name
+    """
+    try:
+        kem = liboqs.OQS_KEM_new(ct.create_string_buffer(alg_name.encode()))
+        if(kem.contents):
+            liboqs.OQS_KEM_free(kem)
+            return True
+    except ValueError:
+        pass
+    return False
+
+
+_KEM_alg_ids = [liboqs.OQS_KEM_alg_identifier(i) for i in range(liboqs.OQS_KEM_alg_count())]
+_supported_KEMs = [i.decode() for i in _KEM_alg_ids]
+_enabled_KEMs = [i for i in _supported_KEMs if is_KEM_enabled(i)]
+
+
+def get_enabled_KEM_mechanisms():
+    """Returns the list of enabled KEM mechanisms."""
+    return _enabled_KEMs
+
+def print_enabled_KEM_mechanisms():
```

Comment:
Do we need to have print\_enabled\_KEM\_mechanisms? Doesn't seem like a "fundamental" method in the class.

Reply:
Depends how we want to use the Python wrapper. Given Python's interactive nature; it might be useful to have such functions to print out info about schemes

Reply:
I want to replace this with "pprint()", the built in pretty printer. If there's a need to encapsulate the enabled kem/sigs in another data structure, magic internal methods can be overridden to print what is needed.

Reply:
Fixed! Some missing data accessors have been added to prevent direct access of internal vars.